<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnAClass</class>
 <widget class="QMainWindow" name="QnAClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>721</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Historic</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QnA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Curlz MT</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>All Questions:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listQuestions">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Curlz MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>The answers for the clicked question:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="answersList">
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Curlz MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ask your question here:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addQ_lineEdit">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="questionButton">
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Post Question</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Curlz MT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Answer a question here:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="answerButton">
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Post Answer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>Old English Text MT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QnA.qrc"/>
 </resources>
 <connections/>
</ui>
